src/core/client/admin/components/UserStatus/BanModal.tsx

```diff
@@ -307,15 +337,23 @@ const BanModal: FunctionComponent<Props> = ({
                         Cancel
                       </Button>
                     </Localized>
-                    <Localized id="community-banModal-updateBan">
-                      <Button
-                        type="submit"
-                        ref={lastFocusableRef}
-                        disabled={disableForm}
-                      >
-                        Save
-                      </Button>
-                    </Localized>
+                    {isMultisite && (
```

Comment:
This is somewhat of a personal preference, but I find it clearer to do 
```
isMultisite ? (<Button></Button>) : (<Button></Button>)
```

I also think it could make sense and be cleaner here to just create a separate component called something like `BanModalSubmitButton` here and then determine what text and localization id to use in that component based on whether it's multisite and what the user's ban status is, since those are the main differences. Then the `createSingleSiteSubmitButton` logic could move into that component too.

Reply:
Great suggestion, refactoring now